doc/contributing/hw/comportability/README.md

```diff
@@ -462,62 +462,137 @@ Here is an example specification:
 
 ## Interrupt Handling
 
-Interrupts are critical and common enough to attempt to standardize across the project.
-Where possible (exceptions for inherited IP that is too tricky to convert) all interrupts shall have common naming, hardware interface, and software interface.
-These are described in this section.
+Interrupts are critical and common enough to standardize across the project.
+Where possible (exceptions for inherited IP that is too tricky to convert) all interrupts shall have common naming, hardware interfaces, and software interfaces.
 
-### Event Type Interrupt
+### Conventions
 
-Event type interrupts are latched indications of defined peripheral events that have occurred and not yet been addressed by the local processor.
-All interrupts are sent to the processor as active-high level (as opposed to edge) interrupts.
-Events themselves can be edge or level, active high or low, as defined by the associated peripheral.
-For instance, the GPIO module might detect the rising or falling edge of one its input bits as an interrupt event.
+Currently, all wired CIP interrupts in OpenTitan use active-high level-triggering.
+(An alternate standard is the use of edge-triggered interupts, which are not described here, but may be supported in a future release.)
 
-The latching of the event is done by the auto-generated register file as described below.
-The clearing of the event is done by a processor write when the handling of the event is completed.
-The waveform below shows the timing of the event occurrence, its latched value, and the clearing by the processor.
-More details follow.
+![OpenTitan System Interrupt Architecture](ot_interrupt_arch.svg)
+
+Interrupts sent to the processor are aggregated by a platfrom level interrupt controller (PLIC).
+Within that logic there may be another level of control for enabling, prioritizing, and enumeration.
+Specification of this control is defined in the rv_plic documentation of the corresponding toplevel design.
+
+### Defining Interrupts
+
+The `.hjson` [configuration file](#configuration-file) for a peripheral specifies interrupts in the field 'interrupt_list'.
+See the [example](#configuration-information-in-the-file) file above for one possible use of this schema.
+For each entry in 'interrupt_list', the following keys are defined:
+
+Key | Kind | Type | Description of Value
+--- | ---- | ---- | --------------------
```

Comment:
Is it worth adding some spaces and hyphens to align stuff here? (I notice that we've done so in the next table)